### DISCUSSION TITLE
How to draw SimilarityMaps using MACCS fingerprint?

### DISCUSSION THREAD

**Discussion by swapchavan** [rdkit/rdkit#5119] [Ideas]
Hi,
I would like to draw similarity maps using MACCS fingerprint. I couldn't found anyway to get "bitInfo" information which we get for Morgan fingerprint using 
```
info = {}
rdMolDescriptors.GetMorganFingerprintAsBitVect(mol, 2, nBits=2048, useFeatures=False, bitInfo=info)
```
If "bitinfo" is not valid for MACCS fingerprints in that case how can I draw similarity maps using MACCS fingerprints?
Any help will be deeply appreciated!

**Comment by greglandrum** [comment]
The contributions to the MACCS fingerprints cannot be mapped back to individual atoms without making a bunch of assumptions, so you can't do similarity maps with them.
